commit 77536da3dea5af4d1859e4e754f07f47cf8d7d4c
Author: Florian Weimer <fweimer@redhat.com>
Date:   Fri Jun 24 19:38:14 2022 +0200

    resolv/tst-resolv-noaaaa: Support building for older C standards
    
    This avoids a compilation error:
    
    tst-resolv-noaaaa.c: In function 'response':
    tst-resolv-noaaaa.c:74:11: error: a label can only be part of a statement and a declaration is not a statement
               char ipv4[4] = {192, 0, 2, i + 1};
               ^~~~
    tst-resolv-noaaaa.c:79:11: error: a label can only be part of a statement and a declaration is not a statement
               char *name = xasprintf ("ptr-%d", i);
               ^~~~

diff --git a/resolv/tst-resolv-noaaaa.c b/resolv/tst-resolv-noaaaa.c
index 56b25f88a58ad286..6e0c6b6fb809e245 100644
--- a/resolv/tst-resolv-noaaaa.c
+++ b/resolv/tst-resolv-noaaaa.c
@@ -71,14 +71,18 @@ response (const struct resolv_response_context *ctx,
       switch (qtype)
         {
         case T_A:
-          char ipv4[4] = {192, 0, 2, i + 1};
-          resolv_response_add_data (b, &ipv4, sizeof (ipv4));
+          {
+            char ipv4[4] = {192, 0, 2, i + 1};
+            resolv_response_add_data (b, &ipv4, sizeof (ipv4));
+          }
           break;
 
         case T_PTR:
-          char *name = xasprintf ("ptr-%d", i);
-          resolv_response_add_name (b, name);
-          free (name);
+          {
+            char *name = xasprintf ("ptr-%d", i);
+            resolv_response_add_name (b, name);
+            free (name);
+          }
           break;
         }
       resolv_response_close_record (b);
